notebooks/GPU_Cluster_Sizing_with_NeMo_Agent_Toolkit.ipynb

```
@@ -0,0 +1,1164 @@
+    "    <b>NOTE:</b> You can bring your own config file! Simply replace <b>source_config</b> below with a path to your uploaded config file in the <b>NeMo-Agent-Toolkit</b> repo. \n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "source_config = \"examples/evaluation_and_profiling/simple_calculator_eval/configs/config-sizing-calc.yml\"\n",
+    "\n",
```

Ruff (0.12.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)